File: csharp/Windows/list1/Backup/list1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace list1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            listBox1.Items.Add(textBox1.Text);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            listBox1.Items.Remove(textBox1.Text);
        }

        private void button4_Click(object sender, EventArgs e)
        {
            String str;
            label1.Text = "";
            foreach (Object s in listBox1.SelectedItems)
            {
                str= s as String;
                label1.Text+=str+"\n";
               
            }
        }

        private void button3_Click(object sender, EventArgs e)
        {
            label2.Text = comboBox1.SelectedItem.ToString();
        }
    }
}

File: VB.NET/windows/list1/Backup/list1/Form1.vb
Public Class Form1

    Private Sub ListBox1_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ListBox1.SelectedIndexChanged
        Label1.Text = ListBox1.SelectedItem
    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        Dim s As String

        For Each s In ListBox1.SelectedItems
            Label1.Text += vbNewLine + s
        Next
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Dim a As String
        a = InputBox("Enter a Value")
        If ListBox1.Items.Contains(a) Then
            ListBox1.Items.Remove(a)
        Else
            MsgBox("Invalid Input")
        End If


    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim a As String
        a = InputBox("Enter a Value")
        ComboBox1.Items.Add(a)
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        Dim a As Integer
        a = InputBox("Enter a Value")
        If a >= 0 And a < ListBox1.Items.Count Then
            ListBox1.Items.RemoveAt(a)
        Else
            MsgBox("Invalid Input")
        End If

    End Sub

    Private Sub LinkLabel1_LinkClicked(ByVal sender As System.Object, ByVal e As System.Windows.Forms.LinkLabelLinkClickedEventArgs) Handles LinkLabel1.LinkClicked
        System.Diagnostics.Process.Start("D:/bsuresh/personals/7 wonders/img12.jpg")
    End Sub
End Class
